docs/userguide/networking/get-started-overlay.md

```diff
@@ -314,11 +311,15 @@ to have external connectivity outside of their cluster.
 
 You can try starting a second network on your existing Swarm cluster using Docker Compose.
 
-1. Log into the Swarm master.
+1. Install Docker Compose. If you install on a Mac or Windows with Docker
+  Toolbox, you can select Docker Compose during installation.
 
-2. Install Docker Compose.
+2. Change your environment to the Swarm master.
 
-3. Create a `docker-compose.yml` file.
+		$ eval $(docker-machine env --swarm mhs-demo0)
+
+3. Create a `docker-compose.yml` file in a directory named `counter`. Compose
```

Comment:
The sentence starting with Compose should appear under the step as it is explanatory. 

![image](https://cloud.githubusercontent.com/assets/1347209/11065155/944b43ac-8776-11e5-93cf-1c17f7c20a1c.png)